File: snippets/python/facet.py
from qdrant_client import QdrantClient, models

client = QdrantClient(url="http://localhost:6333")

client.facet(
    collection_name="{collection_name}",
    key="my-payload-key",
    facet_filter=models.Filter(must=[models.Match("color", "red")]),
    limit=10,
)

File: snippets/csharp/facet.cs
using Qdrant.Client;
using Qdrant.Client.Grpc;

var client = new QdrantClient("localhost", 6334);

await client.FacetAsync(
	"{collection_name}",
	filter: MatchKeyword("color", "red"),
	key: "my-payload-key",
	limit: 10
);